dumper/app/src/test/java/com/google/edwmigration/dumper/application/dumper/connector/oracle/AbstractOracleConnectorTest.java

```diff
@@ -34,6 +34,39 @@ public class AbstractOracleConnectorTest {
   private final ConnectorArguments arguments = mock(ConnectorArguments.class);
   private final AbstractOracleConnector connector = new OracleMetadataConnector();
 
+  @Test
+  public void getDefaultFileName_notForAssessment_success() {
+    OracleLogsConnector logs = new OracleLogsConnector();
+    OracleMetadataConnector metadata = new OracleMetadataConnector();
+    OracleStatsConnector stats = new OracleStatsConnector();
+
+    assertEquals("dwh-migration-oracle-logs.zip", logs.getDefaultFileName(false));
```

Comment:
I would suggest adding a parameter name for the `false` argument.

```
logs.getDefaultFileName(/* isAssessment= */ false)
```

Reply:
Done, I added the comments.